File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if(nums.length == 0 || nums.length == 1){
        return nums.length;
    }
    
    var j = 0;
    
    for(let i = 0; i < nums.length - 1; i++){
        if(nums[i] != nums[i+1]){
            nums[j++] = nums[i]
        }
    }
    nums[j++] = nums[nums.length - 1]
    
    return j
}

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        leng = len(nums)
        if leng == 0 or leng == 1:
            return leng
        
        j = 0;
        
        for i in range(0, leng - 1):
            if(nums[i] != nums[i+1]):
                
                nums[j] = nums[i]
                j+=1
               
        nums[j] = nums[leng - 1]
        j+=1 
        return j